.gitignore

```diff
@@ -26,4 +26,6 @@ cover
 htmlcov
 dist
 build
+
 proxy/public
+docs/_build
```

Comment:
Why did you do this? It seems like you're duplicating tox's effort. It's a bad idea.

Reply:
I used `python -m sphinx` and I ended up with `docs/_build`.  May be we should update that to use `tox`

Reply:
Well, the idea is that you don't do this because tox already has that wrapped and managed under `.tox/`.